low-level/framework/utils/service_logging.py

```diff
@@ -19,130 +19,23 @@
  ****************************************************************************
 """
 
-import logging.handlers
-import time
+import sys
 import os
+from cortx.utils.log import Log
 
+def init_logging(dcs_service_name, file_path, log_level="INFO"):
+    """Initialize logging for SSPL component."""
+    try:
+        Log.init(service_name=dcs_service_name, log_path=file_path, level=log_level)
```

Comment:
what about log rotation config ? What are the defaults, if any exists, does it server SSPL needs ?

Reply:
Log rotation is handled by cortx-utils itself.
Default config in cortx-utils is,

> backup_count=10,
file_size_in_mb=10,
max_bytes = file_size_in_mb * 1024 * 1024

most of the components like HA using this default config only, I think for SSPL also this will suffice

Reply:
yes it suffice. but still add comments about default config around this code and also mention its recommended to refer to latest code for confirming config. that may help in future